examples/platform-specific/openmote-b/hello-world/Makefile

```diff
@@ -0,0 +1,8 @@
+CONTIKI_PROJECT = hello-world
+all: $(CONTIKI_PROJECT)
+
+MAKE_MAC = MAKE_MAC_TSCH
+MODULES += os/services/shell
+
+CONTIKI = ../../../..
```

Comment:
No need to define CONTIKI in Makefiles any longer.

Reply:
I didn't know that - just took an example from other examples, all of which contain the definition of CONTIKI.